packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -12,14 +12,39 @@ export interface InitializeNodeEnvironmentOptions extends InitializerOptions {
     processOptions?: Pick<SpawnOptions, 'cwd' | 'shell' | 'env' | 'stdio' | 'windowsHide'>;
 }
 
+/**
+ * Detailed info about process exit.
+ */
+export interface ProcessExitDetails {
+    /**
+     * The exit code if the child exited on its own
+     */
+    exitCode: number | null;
+
+    /**
+     * The signal by which the child process was terminated
+     */
+    signal: NodeJS.Signals | null;
+
+    /**
+     * The last output process sent to stderr stream
+     */
+    lastSeenError: string | null;
```

Comment:
Isn't it the same as `lastError`? What does `seen` change?

Reply:
renamed